ambry-test-utils/src/main/java/com/github/ambry/utils/HelixControllerManager.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
Is this needed, or even correct?  Nearly all our javadocs have the two star convention.

Reply:
The single star is more correct according to intellij since the license comment is not a javadoc, just a block comment. When the two stars are present it gives a warning "Dangling javadoc comment".